src/System Application/App/Entity Text/src/EntityTextImpl.Codeunit.al

```diff
@@ -472,4 +473,5 @@ codeunit 2012 "Entity Text Impl."
         TelemetryCompletionExtraTextTxt: Label 'The completion contains a Translation or Note section.', Locked = true;
         TelemetryPromptManyFactsTxt: Label 'There are %1 facts defined, they will be limited to %2.', Locked = true;
         TelemetryNoAuthorizationHandlerTxt: Label 'Entity Text authorization was not set.', Locked = true;
+        TelemetryGenerationRetryTxt: Label 'Retrying text generation, attempt: %1', Locked = true;
```

Comment:
It is always sending this telemetry even on the first attempt.
Maybe change the text to something like `Text generation attempt: %1`